Sources/AppStoreConnectCLI/Commands/Users/Invitations/InviteUserCommand.swift

```diff
@@ -26,30 +26,20 @@ struct InviteUserCommand: CommonParsableCommand {
     @Argument(help: "The user invitation recipient's last name.")
     var lastName: String
 
-    @Option(parsing: .upToNextOption, help: "Assigned user roles that determine the user's access to sections of App Store Connect and tasks they can perform.")
-    var roles: [UserRole]
-
-    @Flag(help: "Indicates that a user has access to all apps available to the team.")
-    var allAppsVisible: Bool
-
-    @Flag(help: "Indicates the user's specified role allows access to the provisioning functionality on the Apple Developer website.")
-    var provisioningAllowed: Bool
-
-    @Option(parsing: .upToNextOption,
-            help: "Array of bundle IDs that uniquely identifies the apps.")
-    var bundleIds: [String]
+    @OptionGroup()
+    var userInfo: UserInfoArguments
 
     public func run() throws {
         let service = try makeService()
 
-        if allAppsVisible {
+        if userInfo.allAppsVisible {
             try inviteUserToTeam(by: service)
             return
         }
 
-        if !bundleIds.isEmpty {
+        if !userInfo.bundleIds.isEmpty {
```

Comment:
IBA code style preference is to avoid `!` (not) operators in conditionals.  Instead use `== false` or create a `isNotEmpty` helper.